python/pip.bzl

```diff
@@ -42,9 +44,9 @@ def pip_install(requirements, name = "pip", **kwargs):
     )
     ```
 
```

Comment:
```suggestion
Note that this convenience comes with a cost. Every target in the BUILD file which loads the requirements helper in this way will cause an eager-fetch of all the pip dependencies, even if no python targets are requested to be built. In a multi-language repo, this may cause developers to fetch dependencies they don't need, so consider using the long form for imports if this happens.
```